cfpb/consumerfinance.gov
# Remove `UnescapedRichTextBlock`

This PR removes `UnescapedRichTextBlock` from `TextIntroduction`, `Well`, and `FullWidthText`, and removes all tests and support code around it. This block was put in place of Wagtail's built-in `RichTextBlock` until we could remediate SVGs in our rich text field. That work is now nearly complete. 

This PR also edits the migrations that reference `UnescapedRichTextBlock` and replace it with `RichTextBlock`, and makes the initial migration that replaced `RichTextBlock` with `UnescapedRichTextBlock` a noop.

## How to test this PR

1. Attempt to enter the contents of an SVG into the rich text editor, i.e. 
 
   ```<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1200" class="cf-icon-svg"><path d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm-49.7 234.6c0-27.6 22.4-50 50-50s50 22.4 50 50v328.6c0 27.6-22.4 50-50 50s-50-22.4-50-50V339.8zm50 582.5c-39.6 0-71.7-32.1-71.7-71.7s32.1-71.7 71.7-71.7S572 811 572 850.6s-32.1 71.7-71.7 71.7z"></path></svg>```

2. Observe that the preview does not render it as an SVG but rather as the text that was entered:
<img width="295" alt="image" src="https://github.com/cfpb/consumerfinance.gov/assets/10562538/8f5c473b-1708-494c-bcde-01930efcd4e2">

3. Rejoice.

## Checklist
